src/towncrier/test/test_build.py

```diff
@@ -121,11 +121,21 @@ def test_in_different_dir_dir_option(self, runner):
         self.assertEqual(0, result.exit_code)
         self.assertTrue((project_dir / "NEWS.rst").exists())
 
+    @with_project()
+    def test_traverse_up_to_find_config(self, runner):
+        """
+        If the current directory doesn't contain the configuration file, Towncrier
+        should traverse up the directory tree until it finds it.
```

Comment:
Very minor comment - https://jml.io/test-docstrings/

```suggestion
        If the current directory doesn't contain the configuration file, Towncrier
        will traverse up the directory tree until it finds it.
```
```suggestion
        When the current directory doesn't contain the configuration file, Towncrier
        will traverse up the directory tree until it finds it.
```